docs/topics/request-response.rst

```diff
@@ -363,26 +363,28 @@ are some special keys recognized by Scrapy and its built-in extensions.
 
 Those are:
 
-* :reqmeta:`dont_redirect`
-* :reqmeta:`dont_retry`
-* :reqmeta:`handle_httpstatus_list`
-* :reqmeta:`handle_httpstatus_all`
-* :reqmeta:`dont_merge_cookies`
+
+* ``ftp_password`` (See :setting:`FTP_PASSWORD` for more info)
```

Comment:
These `ftp_…` entries should not go before `bindaddress`.

Reply:
Have changed the location of `ftp`

Reply:
Their new location is not right either. They should be between `download_timeout` and `handle_httpstatus_all`.

Reply:
Have made changes, please look at it.

Reply:
Have made changes, please look at it